web/modules/custom/geolayer/modules/map/farm_map.permissions.yml

```diff
@@ -0,0 +1,2 @@
+administer farm map:
+  title: 'Administer farmOS map'
```

Comment:
is it a farmOs map?

Reply:
@pcambra yes, now it has been refactored and renamed to `Geolayer map` everywhere